tools/gfx-unit-test/imported.slang

```diff
@@ -0,0 +1,8 @@
+struct TestFunction
+            {
+                void simpleElementAdd(RWStructuredBuffer<float> buffer, uint index)
+                {
+                    var input = buffer[index];
+                    buffer[index] = input + 1.0f;
+                }
+            };
```

Comment:
The indentation in these files is a little weird.

Reply:
Side effect of all the shader file writes directly from the test. Probably the extra indentation in the test to align those lines.

Reply:
All the files with weird indentation are the ones that are constantly being overwritten at various points in the shader cache tests, so I'm going to opt to not bother in favor of having the piped code in the test look nicer.